.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "C# (.NET)",
+    // Use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    // Configure tool-specific properties
+    "customization": {
+        "vscode": {
+            "extensions": [
```

Comment:
Missing the most important extension: https://marketplace.visualstudio.com/items?itemName=ZeroCInc.slice :)

Reply:
Added